resources/gin/train/tracknet_v2_1_classifier_train.cfg

```diff
@@ -14,21 +14,17 @@ experiment.random_seed = 42
 
 
 ### data ###
-TrackNetV2_1DataLoader.batch_size = 32
-TrackNetV2_1DataLoader.train_dataset = @train/TrackNetV2_1Dataset
-TrackNetV2_1DataLoader.valid_dataset = @valid/TrackNetV2_1Dataset
-TrackNetV2_1DataLoader.max_size = 1000000
+EventWiseSplitDataLoader.batch_size = 32
+EventWiseSplitDataLoader.train_dataset = @train/ClassifierDataset
+EventWiseSplitDataLoader.valid_dataset = @valid/ClassifierDataset
+EventWiseSplitDataLoader.max_size = 100000
 
 # dataset
-TrackNetV2_1Dataset.input_dir = 'output/cgem_t_tracknet_2_1'
-TrackNetV2_1Dataset.path_to_checkpoint = 'lightning_logs/11_11_epoch=190.ckpt'
-TrackNetV2_1Dataset.model_input_features = 3
+ClassifierDataset.input_dir = 'output/cgem_t_tracknet_2_1'
```

Comment:
I suggest to rename the original class to `TracknetClassifierDataset` or something similar

Reply:
changed to TrackNetClassifierDataset